include/seqan3/io/sequence_file/input.hpp

```diff
@@ -800,6 +815,28 @@ class sequence_file_input
  * \relates seqan3::sequence_file_input
  * \{
  */
+
+//!\brief Deduces the sequence input file type from the stream and the format.
+template <IStream2                           stream_type,
+          SequenceFileInputFormat            file_format>
+sequence_file_input(stream_type & stream,
+                    file_format const &)
+    -> sequence_file_input<typename sequence_file_input<>::traits_type,       // actually use the default
+                           fields<field::SEQ, field::ID, field::QUAL>,        // default field ids.
```

Comment:
```suggestion
                           typename sequence_file_input<>::selected_field_ids, // actually use the default
```

check the other ones below too

Reply:
This was not resolved?